### ISSUE TITLE
Some judicial branch domains remain in the accessibility data

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#534]
As mentioned in #501, some judicial branch domains are accidentally being mistaken for executive branch domains.  

It's happening upstream from the creation of the DAP section (see https://pulse.cio.gov/analytics/domains/#q=supreme), as well as HTTPS  (see the [HTTPS CSV export](https://pulse.cio.gov/data/domains/https.csv)).  It's occurrence in the `a11y` section is as a result, since I'd pointed to the DAP section as the universe of domains (live, non-redirecting, executive-only) that we could use as a list of domains.  

To debug this (and to improve the `a11y` workflow), we should trace the workflow back. 

@konklone - Do you know where the DAP and HTTPS datasets derive their 'branch' data from?  It doesn't appear to be in [this root dataset](https://github.com/GSA/data/blob/gh-pages/dotgov-domains/current-federal.csv)?

**Comment by konklone** [comment]
@gbinal It's fixed for DAP: https://pulse.cio.gov/analytics/domains/#q=supreme This was fixed in https://github.com/18F/pulse/pull/515.

And HTTPS explicitly includes legislative and judicial domains, since it's a non-controversial best practice for them as well, so they are not excluded.

However, `a11y` is still showing the supremecourt.gov domain:
https://staging.pulse.cio.gov/accessibility/domains/#q=supreme

The `a11y` workflow uses a totally separate data pipeline. I believe @adelevie ported over the logic in #515 to properly exclude all legislative and judicial domains, but it looks like something in that pipeline is not working.

I renamed the ticket to scope it to the accessibility data only.

cc @nickbristow
